app/build.gradle

```diff
@@ -0,0 +1,178 @@
+apply plugin: 'com.android.application'
+
+configurations.all {
+    resolutionStrategy {
+        force "com.android.support:support-annotations:${supportLibraryVersion}"
+    }
+}
+
+dependencies {
+    /// dependencies for app building
+    implementation project(':owncloud-android-library:owncloud-android-library')
```

Comment:
Ok, we have discussed about this in the library PR - https://github.com/owncloud/android-library/pull/180#discussion_r241313652 - and maybe we have not understood properly each other.

Since in the app we are going to have `app` module, if we are consistent with the naming, we should use `lib` in the library instead of `owncloud-android-library`.

Reply:
You already did it.
This would be my recommendation

![image](https://user-images.githubusercontent.com/3314607/50093952-4dd54480-0212-11e9-91aa-bf09d9bbf9ab.png)

and this is the diff 
![image](https://user-images.githubusercontent.com/3314607/50093915-3302d000-0212-11e9-9229-e217d710ff6c.png)